clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -257,7 +266,10 @@ public void handleReconnect(ClientDescriptor clientDescriptor, byte[] extendedRe
       throw new AssertionError("Client "+ clientDescriptor +" trying to reconnect is not connected to entity");
     }
     clientState.attach();
-    Set<String> cacheIds = reconnectDataCodec.decode(extendedReconnectData);
+    ReconnectData reconnectData = reconnectDataCodec.decode(extendedReconnectData);
+    clientIdMap.put(clientDescriptor, reconnectData.getClientId());
+    invalidIds.add(reconnectData.getClientId());
```

Comment:
This call and the one above make a logical group, duplicated elsewhere - extract method